main/python-numpy/template.py

```diff
@@ -101,7 +101,10 @@ def post_install(self):
 @subpackage("python-numpy-tests")
 def _(self):
     self.subdesc = "tests"
-    self.depends += ["python"]
+    self.depends = [
+        *self.parent.checkdepends,
```

Comment:
this can be just checkdepends...

Reply:
no i mean

you can write just `checkdepends` instead of `self.parent.checkdepends`